src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -301,6 +303,22 @@ private ContextMenu updateContextMenu(ContextMenu contextMenu) {
         return contextMenu;
     }
 
+    private ObservableList<TurboIssue> applyCountQualifierToIssueList(ObservableList<TurboIssue> issueList,
```

Comment:
I'm not sure this is the right place to put this -- we want to try and keep logic out of the UI. By the time the issues get to the UI to be shown, all the effects of filters should already have been applied. You might want to look into how `sort` is implemented, and do something similar for this.